webpack.config.js

```diff
@@ -0,0 +1,25 @@
+const path = require("path");
+
+module.exports = {
+  mode: "development",
```

Comment:
Will we add the production config later?

Reply:
yup!